Asteroid.js

```diff
@@ -38,22 +82,59 @@ class Asteroid extends Sprite {
     createRadii() {
         const radii = [];
         for (let i = 0; i < ASTEROID_RADII; i++) {
-            const radius = (1 - Math.random() * 0.5) * this.radius;
+            const radius = (1 - Math.random() * 0.4) * this.radius;
```

Comment:
I adjusted this here to try to improve the general shape of asteroids. It seems to have reduced the relative size of the protuberances.